docs/esp32/quickref.rst

```diff
@@ -833,9 +833,9 @@ The RMT is ESP32-specific and allows generation of accurate digital pulses with
     import esp32
     from machine import Pin
 
-    r = esp32.RMT(0, pin=Pin(18), clock_div=8)
-    r   # RMT(channel=0, pin=18, source_freq=80000000, clock_div=8)
-    # The channel resolution is 100ns (1/(source_freq/clock_div)).
+    r = esp32.RMT(pin=Pin(18), resolution_hz=10000000)
+    r   # RMT(pin=18, source_freq=80000000, resolution_hz=10000000, clock_div=8)
```

Comment:
`clock_div` is no longer printed, so can be removed from here

Reply:
Fixed by https://github.com/micropython/micropython/pull/16293/commits/cc7fe8c2172c3c41e122ba4da1da30e3def1413c